src/main/java/seedu/programmer/model/student/LabNum.java

```diff
@@ -1,11 +1,14 @@
 package seedu.programmer.model.student;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.programmer.commons.util.AppUtil.checkArgument;
 
 /**
  * Represents a student's name in the ProgrammerError.
  */
 public class LabNum {
+    public static final String MESSAGE_CONSTRAINT =
+            " Lab number should be between 0 and 13 (inclusive)";
```

Comment:
Repeating MESSAGE_CONSTRAINT. A similar message constraint already exists in Lab, don't need to create a new one. You can just refer to the previous one.

Reply:
Ok have changed in the latest commit!